website/docs/guides/best-practices/how-we-mesh/mesh-1-intro.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Intro to dbt Mesh"
+description: Getting started with dbt Mesh patterns
+hoverSnippet: Learn how to get started with dbt Mesh
+---
+
+## What is dbt Mesh?
+
+Organizations of all sizes rely upon dbt to manage their data transformations, from small startups to large enterprises. At scale, it can be challenging to coordinate all the organizational and technical requirements demanded by your stakeholders within the scope of a single dbt project. To date, there also hasn't been a first-class way to effectively manage the dependencies, governance, and workflows between multiple dbt projects.
+
+Regardless of your organization's size and complexity, dbt should empower data teams to work independently and collaboratively; sharing data, code, and best practices without sacrificing security or autonomy. dbt Mesh provides the tooling for teams to finally achieve this.
+
+dbt Mesh is not a single product: it is a pattern enabled by a convergence of several features in dbt:
+
+- **[Cross-project references](/docs/collaborate/govern/project-dependencies#usage)** - this is the foundational feature that enables the multi-project deployments. `{{ ref() }}`s now work across dbt Cloud projects on Enterprise plans.
+- **[dbt Explorer](/docs/collaborate/explore-projects)** - dbt Cloud's metadata-powered documentation platform, complete with full, cross-project lineage.
+- **Governance** - dbt's new governance features allow you to manage access to your dbt models both within and across projects.
+  - **[Groups](/docs/build/groups)** - groups allow you to assign models to subsets of models within a project.
```

Comment:
I found this a little confusing. Is this also correct? If not, can you reword to clarify? Also updating the link. 
```suggestion
  - **[Groups](/docs/collaborate/govern/model-access#groups)** - groups allow you to assign models to subsets within a project.
```